### ISSUE TITLE
Connect to remote socket with SSH?

### ISSUE THREAD

**Issue by bajax** [sequelpro/sequelpro#2120]
I know there are reasons that it doesn't currently support this, but I think it'd be extremely useful to have the ability to connect to a socket-based SQL instance over SSH.  

At our shop, we grant as few privileges as possible to accounts bound to network interfaces, and only grant risky privileges to accounts on the local socket.  I'm not 100% sure why, but HeidiSQL and MySQL Workbench are able to do it without any trouble (possibly by just invoking the CLI client on the remote end instead of 'real' tunneling) but I'd love to be able to do this on Sequel Pro as well.

**Comment by dmoagx** [comment]
I spent quite some time comparing Sequel Pros SSH support to other products (including HeidiSQL and MySQL Workbench).
I did not find any case where Sequel Pro could not connect, but another tool could.

If you know such a setup please provide detailed steps to reproduce.

**Comment by bajax** [comment]
I found out later there was something I had overlooked in the config.  The server that it "worked" on was really listening on all interfaces.  It was a while ago so I may be wrong on the exact details, but if I remember right the other client used "localhost" for the other side of the SSH tunnel, which resolved to 127.0.0.1 before it ever got to mySQL, stripping it of its special meaning.  We got it fixed and now, as expected, the other clients correctly (heh) don't work either.
